srtcore/common.h

```diff
@@ -589,12 +589,42 @@ class InvertedGuard
 class CSeqNo
 {
 public:
+
+   /// This behaves like seq1 - seq2, in comparison to numbers,
+   /// and with the statement that only the sign of the result matters.
+   /// That is, it returns a negative value if seq1 < seq2,
+   /// positive if seq1 > seq2, and zero if they are equal.
+   /// The only tolerable use of this function is only when you
```

Comment:
```suggestion
   /// The only useful application of this function is when you
```

Reply:
I'd prefer "correct application". The all other uses are not just "useless", but erroneous.